src/pages/main_layout.cr

```diff
@@ -110,11 +110,11 @@ abstract class MainLayout
 
     ## What does Lucky look like?
     
-    In Lucky we generate separate action classes with names that describe the route.
-    The resulting file maps routing path definitions to a response block.
+    Lucky will generate **action classes** from class names that describe the route and determine
```

Comment:
I think I'm a  bit confused about Lucky generating an action class.  Lucky uses the class, but does not generate one. Maybe this would be more accurate:

> Lucky uses a class for handling HTTP requests and responses. The class determines which route and parameters it handles.
>
> Using separate classes allows to provide very [solid](https://thoughtbot.com/blog/designing-lucky-actions-routing) automatic error detection, as well as generation of routing, path, and link helpers and methods.

The last sentence is unchanged. What do you think?

Reply:
I think your sentence is technically ok, of course, only it could convey more what Lucky "is like".

What I had in mind with "Lucky will  generate" was hinting at the generator commands
like `lucky gen.action.browser Users::Index`, which lower the typing burden, something that always seems to come to mind when first reading about having to create separate classes.

Should we maybe explicitly mention the `lucky gen.action`?
Or like this:
> Lucky's generator creates **action classes**  from class names that describe the route and determine

Reply:
Maybe rather than adding that in the description we add a second sentence. 

> Lucky can generate these classes for you with 'lucky gen.action`

That way it's super clear what we're trying to say.

Thoughts?

Reply:
I seem to be missing the cool part: User provided "REST" class name => route path definitions generated (currently through `route` inference).

Reply:
That will likely be removed so I don't think we should make that one of the main things we talk about on the home page. Especially since the code example doesn't use it.

Reply:
Idea:
> Lucky's code generator takes class names that describe a route, e.g. `Api::Users::Show` and creates the classes with corresponding routes. The classes map the routing path definition to a response block.

Reply:
I understand wanting to be super clear and show all the goodies, but this isn't *quite* right yet though since it doesn't generate the corresponding route yet, it uses `route`. But since we don't want to advertise that yet since it may change or be removed. I also don't want to go too into the weeds with generators. To get the best of both worlds we should go with:

> Lucky uses a class for handling HTTP requests and responses. The class determines which route and parameters it handles. Lucky can generate these classes for you with 'lucky gen.action`
>
> Using separate classes allows to provide very solid automatic error detection, as well as generation of routing, path, and link helpers and methods.

This covers that each class handles requests, defines routes, and that you can use a generator to get things done quickly. That covers things that will for sure be in Lucky, even in future versions

Reply:
Ok, I just hope you're still planning to move the default REST class naming routes from `route` into the generator.

Reply:
For now that is the plan. If we remove `route` we will make the generator generate `get "/users/:user_id"` or whatever :) so it is still fast